components/admin_console/system_users/system_users_dropdown/system_users_dropdown.jsx

```diff
@@ -140,13 +140,13 @@ export default class SystemUsersDropdown extends React.PureComponent {
     handleShowDeactivateMemberModal = async (e) => {
         e.preventDefault();
         if (this.shouldDisableBotsWhenOwnerIsDeactivated()) {
-            const {data} = await this.props.actions.loadBots(
+            await this.props.actions.loadBots(
                 Constants.Integrations.START_PAGE_NUM,
-                Constants.Integrations.PAGE_SIZE
+                Constants.Integrations.PAGE_SIZE,
             );
-            if (data) {
-                this.setState({loading: false, showDeactivateMemberModal: true});
-            }
+            this.setState({showDeactivateMemberModal: true});
+        } else {
```

Comment:
Looks like we could do this at the end of if condition as if and else has the same `this.setState({showDeactivateMemberModal: true});` check

Reply:
You're right, good catch.